project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/analysis/ProjectOrFolderAnalyzer.java

```diff
@@ -180,29 +187,30 @@ private void setSnapshotIdAndGroupId(AnalysisConfig config) {
         }
     }
 
-    private void setResultHandler(AnalysisConfig config) {
+    private void setResultHandler(final AnalysisConfig config) {
         config.setResultHandler(new IAnalysisResultHandler() {
             @Override
             public void handleAnalysisResult(final List<AnalysisResult> resultList, final IDexterClient client) {
                 try {
+                    File resultFile = new File(config.getResultFileFullPath()); // DexterAnalyzer.getResultFile(resultList);
+                    Files.createParentDirs(resultFile);
+
                     List<Defect> allDefectList = DexterAnalyzer.getAllDefectList(resultList);
 
-                    int size = allDefectList.size() * 1024;
+                    long size = allDefectList.size() * 1024l;
 
                     StringBuilder msg;
-                    if (size <= Integer.MAX_VALUE)
-                        msg = new StringBuilder(size);
-                    else
+                    if (size > Integer.MAX_VALUE)
```

Comment:
This is intesional decision. StringBuilder contructor can have under Integer.MAX_VALUE.